### PR TITLE
CPR-68 Add new script and README for accessing AWS resources

### PR DESCRIPTION
**Pull request by cairnsp**
Two new files have been added: a script for setting up a Kubernetes service pod and a corresponding README that outlines how to edit and run this script. The new files aim to help users configure their environment and access AWS resources more effectively.

### CODE REVIEW

**Review by vb-moj — COMMENTED** [review]
How are we going to test this without any files?

**Review by markreesmoj — CHANGES_REQUESTED** [review]
Looks like a very useful tool

File: scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+## Accessing AWS Resources via the AWS CLI
+
+### Setting the kubenetes namespace for the environment
+
+Edit the bash script and set the namespace variable to one of:
+
+- hmpps-persons-record-dev
+- hmpps-persons-record-preprod
+- hmpps-persons-record-prod 
+
```
[the remark this answers is not in the document]
Comment (vb-moj) [reply scripts/README.md:11]: Agree.

**Review by vb-moj — COMMENTED** [review]
A good candidate for Show & Tell tomorrow.

File: scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+`$ ./setup-service-pod.bash` 
+
+This will create a k8s service pod that allows access to AWS resources using the AWS Service Account
+associated with the hmpps-person-record-dev/preprod/prod cluster.
+
+The script sets up some convenience environment variables to facilitate testing:
+
+- RDS_INSTANCE_IDENTIFIER
+- COURT_CASE_EVENTS_QUEUE_URL
+- COURT_CASE_EVENTS_DLQ_QUEUE_URL
```
Comment (vb-moj) [inline scripts/README.md:23]: I think this should be CPR Queues, No?

File: scripts/setup-service-pod.bash [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    fi
+}
+
+echo "🔑 Getting Person Record RDS instance from secrets ..."
+secret_json=$(cloud-platform decode-secret -s hmpps-person-record-rds-instance-output -n $namespace --skip-version-check)
+export RDS_INSTANCE_IDENTIFIER=$(echo "$secret_json" | jq -r .data.rds_instance_address | sed s/[.].*//)
+
+echo "🔑 Getting court case events queue url from secrets..."
+secret_json=$(cloud-platform decode-secret -s sqs-cpr-court-case-events-secret -n $namespace --skip-version-check)
+export COURT_CASE_EVENTS_QUEUE_URL=$(echo "$secret_json" | jq -r .data.sqs_queue_url)
```
Comment (vb-moj) [inline scripts/setup-service-pod.bash:36]: Need CPR q here.

**Review by vb-moj — CHANGES_REQUESTED** [review]
Queus need changing.

File: scripts/setup-service-pod.bash [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+echo "🔑 Getting court case events DLQ url from secrets..."
+secret_json=$(cloud-platform decode-secret -s sqs-cpr-court-case-events-dlq-secret -n $namespace --skip-version-check)
+export COURT_CASE_EVENTS_DLQ_QUEUE_URL=$(echo "$secret_json" | jq -r .data.sqs_queue_url)
```
[the remark this answers is not in the document]
Comment (vb-moj) [reply scripts/setup-service-pod.bash:40]: Sorry, confused with the name but could you please change the name?

**Review by markreesmoj — DISMISSED** [review]
👍

**Review by vb-moj — DISMISSED** [review]

**Review by markreesmoj — APPROVED** [review]
👨‍🍳

**Pull request merged**